pkg/kubelet/reason_cache.go

```diff
@@ -40,8 +40,8 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// Reason is the cached item in ReasonCache
-type reasonItem struct {
+// ReasonItem is the cached item in ReasonCache
+type ReasonItem struct {
```

Comment:
Why make thsi public at the same time?  it can still be reasonItem

Reply:
Thanks @thockin
Because the function `Get` is public, it's return value should be public also IMO.
Ref : https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/reason_cache.go#L96

We can't change `Get` to `get`, as it is called in other packages.

Reply:
My point was that it's not public today and you are changing it to public.  Clearly whomever is using it is OK with the type not being public.  It's a little weird, sure, but it's not illegal.

That said it's pretty insignifiicant, so sure.  It's less weird.